tests/integration/DOM-test.js

```diff
@@ -184,3 +185,17 @@ test('Clicking modal overlay', function(assert) {
   });
 
 });
+
+test('Pressing escape', function(assert) {
```

Comment:
[Here is an example of how to use testModal](https://github.com/sir-dunxalot/ember-modals/blob/master/tests/integration/options-test.js#L152)